From 906d70e8e04a8a7f5062053a1f01fb318deb6910 Mon Sep 17 00:00:00 2001
From: Jan Beich <jbeich@FreeBSD.org>
Date: Sun, 31 Oct 2021 07:32:18 +0000
Subject: [PATCH] VA/X11: VAAPI driver mapping for crocus DRI driver

crocus supports Gen4 to Gen7.5, so only i965 can be used.
---
 va/x11/va_x11.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/va/x11/va_x11.c b/va/x11/va_x11.c
index 4d06acd2..9687afaf 100644
--- a/va/x11/va_x11.c
+++ b/va/x11/va_x11.c
@@ -55,6 +55,7 @@ static const struct driver_name_map g_dri2_driver_name_map[] = {
     { "i965",       4, "i965"   }, // Intel i965 VAAPI driver with i965 DRI driver
     { "iris",       4, "iHD"    }, // Intel iHD  VAAPI driver with iris DRI driver
     { "iris",       4, "i965"   }, // Intel i965 VAAPI driver with iris DRI driver
+    { "crocus",     6, "i965"   }, // Intel i965 VAAPI driver with crocus DRI driver
     { NULL,         0, NULL }
 };
 
